src/blocks/gallery-masonry/v1/edit.js

```diff
@@ -0,0 +1,272 @@
+					gridSize: 'lrg',
+				} );
+			}
+		}
+	}, [ gridSize, align, wideControlsEnabled ] );
+
+	useEffect( () => {
+		// Deselect images when deselecting the block.
+		if ( ! isSelected && prevIsSelected ) {
+			setSelectedImage( null );
+		}
+	}, [ prevIsSelected, isSelected ] );
+
+	const onSelectImage = ( index ) => {
+		return () => {
+			if ( selectedImage !== index ) {
+				setSelectedImage( index );
+			}
+		};
+	};
+
+	const onMove = ( oldIndex, newIndex ) => {
+		const newImages = [ ...attributes.images ];
+		newImages.splice( newIndex, 1, attributes.images[ oldIndex ] );
+		newImages.splice( oldIndex, 1, attributes.images[ newIndex ] );
+		setSelectedImage( newIndex );
+		setAttributes( { images: newImages } );
+	};
+
+	const onMoveForward = ( oldIndex ) => {
+		return () => {
+			if ( oldIndex === attributes.images.length - 1 ) {
+				return;
+			}
+			onMove( oldIndex, oldIndex + 1 );
+		};
+	};
+
+	const onMoveBackward = ( oldIndex ) => {
+		return () => {
+			if ( oldIndex === 0 ) {
+				return;
+			}
+			onMove( oldIndex, oldIndex - 1 );
+		};
+	};
+
+	const onRemoveImage = ( index ) => {
+		return () => {
+			const newImages = filter( attributes.images, ( _img, i ) => index !== i );
+			setSelectedImage( null );
+			setAttributes( {
+				images: newImages,
+			} );
+		};
+	};
+
+	/**
+	 * replaceImage is passed to GalleryImage component and is used to replace images
+	 *
+	 * @param {number} index Index of image to remove.
+	 * @param {Object} media Media object used to initialize attributes.
+	 */
+	const replaceImage = ( index, media ) => {
+		const newImages = [ ...attributes.images ];
+		newImages[ index ] = { ...media };
+
+		setAttributes( { images: newImages } );
+	};
+
+	const setImageAttributes = ( index, newAttributes ) => {
+		if ( ! images[ index ] ) {
+			return;
+		}
+		setAttributes( {
+			images: [
+				...images.slice( 0, index ),
+				{
+					...images[ index ],
+					...newAttributes,
+				},
+				...images.slice( index + 1 ),
+			],
+		} );
+	};
+
+	const masonryGalleryPlaceholder = (
+		<>
+			{ ! hasImages ? noticeUI : null }
+			<GalleryPlaceholder
+				{ ...props }
+				gutter={ gutter }
+				icon={ <Icon icon={ icon } /> }
+				label={ __( 'Masonry', 'coblocks' ) }
+			/>
+		</>
+	);
+
+	if ( ! hasImages ) {
+		return masonryGalleryPlaceholder;
+	}
+
+	const innerClasses = classnames(
+		...GalleryClasses( attributes ),
+		sidebarIsOpened, {
+			[ `align${ align }` ]: align,
+			'has-gutter': gutter > 0,
+			'has-lightbox': lightbox,
+		}
+	);
+
+	const masonryClasses = classnames(
+		`has-grid-${ gridSize }`, {
+			[ `has-gutter-${ gutter }` ]: gutter > 0,
+			[ `has-gutter-mobile-${ gutterMobile }` ]: gutterMobile > 0,
+		}
+	);
+
+	const itemClasses = classnames(
+		'coblocks-gallery--item', {
+			[ `coblocks-animate ${ animation }` ]: animation,
+		}
+	);
+
+	return (
+		<>
+			{ isSelected &&
+			<Controls
+				{ ...props }
+			/>
+			}
+			{ isSelected &&
+			<Inspector
+				{ ...props }
+			/>
```

Comment:
I don't think we need the extra spacing here.
```suggestion
			<Inspector { ...props } />
```

Reply:
Fixed the spacing 👍